internal/store/pod_test.go

```diff
@@ -842,6 +844,13 @@ kube_pod_container_status_last_terminated_reason{container="container7",namespac
 					HostIP:    "1.1.1.1",
 					PodIP:     "1.2.3.4",
 					StartTime: &metav1StartTime,
+					Conditions: []v1.PodCondition{
+						{
+							Type:               "Ready",
+							Status:             "True",
+							LastTransitionTime: metav1ReadyTime,
```

Comment:
Can we add a test case when condition is not ready and set the status and LastTransitionTime. e.g. when the metric should not be set then.

Reply:
Sure, I can do it.